apps/jobs/utils.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import status
+from rest_framework.response import Response
+from challenges.models import (
+    ChallengePhase,
+    Challenge,
+)
+from participants.models import (ParticipantTeam,)
+
+
+def get_challenge_object(challenge_pk):
+    try:
+        Challenge.objects.get(pk=challenge_pk)
+    except Challenge.DoesNotExist:
+        response_data = {'error': 'Challenge does not exist'}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+def get_challenge_phase_object(challenge_phase_pk):
```

Comment:
I dont think there is need of `app_name` here since we are already namespacing this functions by keeping them in `utils.py` of each app. That is actually more than enough. So with this anyone wanting to use will write something like

`from apps.challenges.utils import get_challenge_object`

One more thing to keep in mind is that these functions should simply return a object or raise an exception if obejct is not found. In any case I dont think it makes sense to return Response object as that will bring a hassle to the callee of keeping explicit checks of value being returned. This may be debatable @deshraj : your thoughts required here.